__tests__/pages/delegates.tsx

```diff
@@ -151,10 +164,10 @@ describe('Delegate Create page', () => {
     click(closeUndelegateBtn);
 
     // Voting weights are returned to 0 after undelegating
-    const newTotal = await screen.findByText(/Total MKR delegated/);
+    const newTotal = await screen.getByTestId('total-mkr-delegated');
```

Comment:
I think it's better to find something by the text since that's what the user sees. Test ID might not change but it doesn't mean what the user is seeing hasn't changed in some future code change.

Reply:
Actually testIDs are better than text , text is not only limited to one place, data-testids can be unique.  They are more "resilient" to change. A simple change on a copy may made your test unreliable.

Reply:
In tis case there is now 2 components with the same text, one for system info and one for the delegate